src/main.rs

```diff
@@ -43,6 +47,18 @@ fn run() -> Result<()> {
         env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "off"),
     );
 
+    let raw_args: Vec<String> = env::args().collect();
+    if raw_args.len() > 1 && raw_args[1].starts_with("mailto:") {
```

Comment:
Is this meant to be used if you press in a webbrowser on an email address?

Reply:
Yes, see https://github.com/soywod/himalaya/issues/160 and https://github.com/soywod/himalaya/issues/162.

Reply:
That's a neat feature! :D